tests/components/sensor/test_yr.py

```diff
@@ -32,12 +31,9 @@ def test_default_setup(self, betamax_session):
                    return_value=betamax_session):
             with patch('homeassistant.components.sensor.yr.dt_util.utcnow',
                        return_value=now):
-                assert sensor.setup(self.hass, {
-                    'sensor': {
-                        'platform': 'yr',
-                        'elevation': 0,
-                    }
-                })
+                    assert _setup_component(self.hass, 'sensor', {
```

Comment:
Is that really necessary when you only have one _setup_component for each method, and the teardown_method has a  self.hass.stop() ?